python/src/etos_api/routers/lib/kubernetes.py

```diff
@@ -0,0 +1,43 @@
+# Copyright Axis Communications AB.
+#
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generic Kubernetes helpers for all submodules."""
+import logging
+import os
+
+from kubernetes import config
+
+NAMESPACE_FILE = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+LOGGER = logging.getLogger(__name__)
+
+try:
+    config.load_incluster_config()
+except config.ConfigException:
+    try:
+        config.load_config()
+    except config.ConfigException:
+        LOGGER.warning("Could not load a Kubernetes config")
+
+
+def namespace() -> str:
+    """Get current namespace if available."""
+    if not os.path.isfile(NAMESPACE_FILE):
+        LOGGER.warning(
+            "Not running in Kubernetes. Cannot figure out namespace. "
+            "Defaulting to environment variable 'ETOS_NAMESPACE'."
+        )
+        return os.getenv("ETOS_NAMESPACE")
```

Comment:
We should add a "check if set" or similar here and if ETOS_NAMESPACE is not set log a warning, as Kubernetes will then deploy to the 'default' namespace.

Reply:
Added a warning. Should an exception be raised if namespace is undefined? I haven't changed the behavior yet. So, only warning for now.

Reply:
Hrm, not sure if Warning is enough here since we're deleting things. @t-persson thoughts?